packages/web/hooks/contracts.ts

```diff
@@ -65,13 +66,142 @@ export const useHustler = () => {
   );
 };
 
-export const useReleaseDate = (): Date | undefined => {
-  const [releaseDate, setReleaseDate] = useState<Date>();
-  const initator = useInitiator();
+export const useFetchMetadata = () => {
+  const hustler = useHustler();
+  const { provider } = useOptimism();
 
-  useEffect(() => {
-    initator.release().then(date => setReleaseDate(new Date(date.toNumber() * 1000)));
-  }, [initator]);
+  return async function fetchMetadata(id: BigNumber) {
+    const metadata = await hustler.metadata(id);
```

Comment:
we can get the OGs title off the Components contract for now: https://github.com/dopedao/dope-monorepo/blob/master/packages/contracts/src/Components.sol#L386